src/common/s3_adapter.cpp

```diff
@@ -132,6 +132,8 @@ void InitS3AdaptorOptionExceptS3InfoOption(Configuration* conf,
         &s3Opt->bpsReadMB));
     LOG_IF(FATAL, !conf->GetUInt64Value("s3.throttle.bpsWriteMB",
         &s3Opt->bpsWriteMB));
+    LOG_IF(FATAL, !conf->GetBoolValue("s3.useVirtualAddressing",
+        &s3Opt->useVirtualAddressing));
```

Comment:
also need add this config in func InitS3AdaptorOption?

Reply:
InitS3AdaptorOption will reuse InitS3AdaptorOptionExceptS3InfoOption